react/features/base/connection/actions.native.js

```diff
@@ -180,13 +183,13 @@ function _connectionDisconnected(connection, message) {
  *
  * @param {JitsiConnection} connection - The JitsiConnection which was
  * established.
- * @private
  * @returns {{
  *     type: CONNECTION_ESTABLISHED,
  *     connection: JitsiConnection
  * }}
+ * @public
```

Comment:
I hope we don't postpone a decision on this long enough to end up with 2 different coding styles merely out of personal preference.